nomad/job_endpoint.go

```diff
@@ -53,6 +53,27 @@ var (
 type Job struct {
 	srv    *Server
 	logger log.Logger
+
+	// builtin admission controllers
+	mutators   []jobMutator
+	validators []jobValidator
+}
+
+// NewJobEndpoints creates a new job endpoint with builtin admission controllers
+func NewJobEndpoints(s *Server) *Job {
```

Comment:
this is very cool - any plans to make it a feature where you could use external binaries/custom admission controllers without hacking nomad source code?

Reply:
Its certainly been designed with that goal :smile: